roles/remove_stale_lun/README.md

```diff
@@ -0,0 +1,40 @@
+oVirt Remove Stale LUN
+=========
+
+The `remove_stale_lun` role iterates through all the hosts in a data center and remove stale LUN devices from these hosts.
```

Comment:
Please add that the user needs to add ssh key to the hosts if they are not executing the playbook on the engine.